src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.html

```diff
@@ -2,10 +2,10 @@
 <div [ngGrid]="gridConfig"
      (onDragStop)="OnDragStop()"
      class="dot-edit-layout-grid__grid">
-
-    <div style="background: #b2b2b2; border: 1px solid red;"
-         *ngFor="let container of gridContainers; let i = index"
-         [(ngGridItem)]="container.config">Index: {{i}}
-        <a (click)="removeContainer(i)">Remove</a>
-    </div>
+    <dot-container-selector *ngFor="let container of gridContainers; let i = index"
+        [(ngGridItem)]="container.config"
+        [selectedContainersList]="container.containers"
```

Comment:
This param name makes no sense... should be something like: ```value```.

Reply:
we will chance the name "selectedContainersList" in the next re-factor.